src/content/structured/components/top-nav/code.mdx

```diff
@@ -240,7 +240,7 @@ export const snippetsCentreAligned = [
   {
     language: "Web component",
     snippet: `<ic-top-navigation app-title="ICDS" status="alpha" version="v0.0.7" content-aligned="center">
-  <svg slot="app-icon" ...></svg>
+  <svg slot="icon" ...></svg>
```

Comment:
This one can stay as app-icon, it's just any slotted svgs within navigation buttons that need to be slot="icon"
```suggestion
  <svg slot="app-icon" ...></svg>
```